code/script.js

```diff
@@ -30,18 +34,114 @@ const showMessage = (message, sender) => {
+  showMessage(`Really feel like watching something from the ${year} `,  'user');
+  showMessage(`${year} you say! Cool, what genre do you feel like?`, 'bot');
+
+  if (year === '1980s') { //fist conditional, different drop down select menus depeneding on users choice of decade
+    inputWrapper.innerHTML = `
+      <select id="select"> 
+        <option value="" selected disabled>Pick a genre</option>
+        <option value="1980's comedy">An 1980's comedy</option>
+        <option value="1980's horror">An 1980's horror</option>
+        <option value="1980's documentary">An 1980's documentary</option>
+      </select>
+    `// this creates a drop-down select menu in the input wrapper 
+  } else if (year === '1990s') {
+      inputWrapper.innerHTML = `
+       <select id="select">
+        <option value="" selected disabled>Pick a genre</option>
+        <option value="1990's comedy">An 1990's comedy</option>
+        <option value="1990's horror">An 1990's horror</option>
+       <option value="1990's documentary">An 1990's documentary</option>
+      </select>
+    `
+  } else {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>Pick a genre</option>
+        <option value="2000's comedy">A 2000's comedy</option>
+        <option value="2000's horror">A 2000's horror</option>
+       <option value="2000's documentary">A 2000's documentary</option>
+      </select>
+    `
+  }
+
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => confirmation(select.value))//this allows user to select from drop down menu leads to next question
+}
+
+const confirmation = (select) => {
+  showMessage(select, 'user');
+  setTimeout(() => showMessage(`Ohh ${select}, fancy! Do you have a Netflix account?`, 'bot'), 1000);
+  
+  inputWrapper.innerHTML = `
+    <button id='yesBtn'>Yes</button>
+    <button id='noBtn'>No</button>
+  `
+
+  document.getElementById('yesBtn').addEventListener("click", () => tooBad('yes'));
+  document.getElementById('noBtn').addEventListener("click", () => tooBad('no'));
+  button.addEventListener('select', () => tooBad(answer));
+
+  //this function leads to two different option depending on whether you click yes or no. 
+  //Yes leads to pick a movie and no to the option to create an account on Netflix, using conditionals
+}
+
+const tooBad = (answer) => { 
+  showMessage(answer, 'user');
+
+  if (answer === 'no') {
+  setTimeout(() => showMessage(`Too bad. This bot is sponsored by Netflix. To get access to all my suggestions, please create an account. See you soon!`, 'bot'), 1000);
+  inputWrapper.innerHTML = `
+    <button id="signUpBtn"><a href="https://www.netflix.com/signup" target="_blank">Create an account</a></button>
+    `
+  } else {
+  setTimeout(() => showMessage(`Awesome. However, despite your wishes, these are the three best movies of all times. So go ahead and pick one. Enjoy!`, 'bot'), 1000);
```

Comment:
see previous comment, `setTimout` could instead be used when `tooBad` is called in order to delay both the question and the input answer buttons